### ISSUE TITLE
View video stream on Raspberry Pi through hardwired connection

### ISSUE THREAD

**Issue by emeurrens** [emeurrens/parking-availability-system#28]
Primary assignee: Evan Tobon

Criteria:
Be able to display the LIVE camera feed on a monitor via a program window or directly to the monitor. Record the live camera feed working and store the result in the Design Prototype folder in the OneDrive.
Be able to change frame rate at which image is captured. See if this can be verified by timing how long it takes for each frame to be ready (e.g. print time it takes for a frame to print and average the times using the average time to find the average frame rate)

**Comment by emeurrens** [comment]
- [x] Got live video feed from TCP, not over hardwired connection, but good enough.  Recording in OneDrive (Assignments > Design Prototype)

- [x] Changed frame rate of camera and confirmed frame rate of camera manually using counters and physically timing with stopwatch

Some **bugs** we encountered:
- For some reason, things are . . . [blue](https://www.youtube.com/watch?v=68ugkg9RePc). Check the OneDrive to actually see what I mean (Assignments > Design Prototype > .mkv file)
- Had to power cycle Pi multiple times (4-5 times within a 2-3 hour period) because it would frequently hang when testing Python code. Signs point to slow I/O; processes waiting on I/O operations. Suspicion is on the SD card, so I ordered a new SD card where we will reinstall and reconfigure the RPi.
